docs/index.md

```diff
@@ -1,21 +1,58 @@
-<div class="marketing-background">
-    <div class="marketing-header">
-        <img src="images/QlikCore-Horizontal-Print-white.svg"/><br>
-    </div>
-    <div class="marketing-text">
-        <h1>Qlik Core Beta</h1>
-        <p>Welcome to Qlik Core Beta! This beta program gives you early access to Qlik Core,
-        our brand-new plug-and-play developer platform for custom analytics that lets you
-        drop the Qlik Associative Engine into your systems today – and act on your insight
-        tomorrow.</p>
-        <p>As you’re aware, beta versions are subject to being less stable than the final
-        version officially released to the general public. Therefore, should you decide to
-        participate in this beta program, we do not recommend you run your existing
-        production workloads using Qlik Core.</p>
-        <p>If you have questions about Qlik Core Beta use our Slack channels. We would
-        love to get your feedback, how you are using Qlik Core Beta and what we can to
-        do make it better.</p>
-        <br>
-        <a href="./docs/home/" class="marketing-text-button">Get Started</a>
-    </div>
-</div>
+# Home
+
+## Beta
```

Comment:
Personal preference: It would be more visually pleasing to remove the Beta chapter. Beta is already mentioned top left.